app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CartsController < ApplicationController
+  def show
+  end
+
+  def add_to_cart
+    @cart_item = CartItem.new(cart_item_params)
```

Comment:
how is this method different from the `cart_items#create`?

Reply:
Now that I'm looking, it's not. Removed it entirely.